### PR TITLE
Integrating REGRID parcels & other map UI updates

### LINKED ISSUE #67: Migrate to REGRID parcels
Switch from self-created, mapbox-hosted, San Antonio parcels to a subscription to REGRID's tileserver.

Even if we don't need national coverage of parcels for a while, it would be nice to have some basic parcel attributes like `address` that will be available from REGRID.

### PR DESCRIPTION
**Pull request by davemfish**
Fixes #67

### CODE REVIEW

File: frontend/src/requests.js [hunk 1]
```diff
@@ -243,6 +243,7 @@ export async function addParcel(sessionID, studyAreaID, parcelID, wkt) {
         session_id: sessionID,
         study_area_id: studyAreaID,
         parcel_id: parcelID,
+        address: address,
```
Comment (dcdenu4) [inline frontend/src/requests.js:246]: Nice!

File: frontend/src/map/map.jsx [hunk 2]
```diff
@@ -158,7 +160,7 @@ const map = new Map({
     scenarioLayerGroup,
   ],
   view: new View({
-    center: [-10984368.72, 3427876.58], // W. San Antonio, EPSG:3857
+    center: [-10956248.2268683, 3427877.42426285], // E. San Antonio, EPSG:3857
```
Comment (dcdenu4) [inline frontend/src/map/map.jsx:163]: Woah, this is going to throw me!

**Review by dcdenu4 — APPROVED** [review]
@davemfish, looks good! I just saw one `console.log` that I wasn't sure you wanted to leave in. I'll approve though and let you merge at your convenience from here.

File: frontend/src/map/map.jsx (hunk 2 above)
Comment (davemfish) [reply frontend/src/map/map.jsx:163]: Hmm, now that the map zoom's to the selected study area, and session's are saved in the browser, with a study area pre-selected if one exists, we could probably change this to just center on the city at a smaller zoom. And in practice that would only happen for a fresh session with no study areas.

**Pull request merged**